maxfw/utils/image_utils_new.py

```diff
@@ -131,7 +183,6 @@ class Grayscale(object):
     """
     
     def __init__(self, num_output_channels=1):
-        assert num_output_channels in [1,3]
```

Comment:
We usually do sanity checks in the `image_functions.py` file, so I have moved this over.